.github/workflows/docker.yml

```diff
@@ -22,6 +22,12 @@ jobs:
         with:
           ruby-version: 2.6.x
       - run: gem install bundler
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
```

Comment:
This is unnecessary - we already install / use buildx

Reply:
@Iristyle 
```This is unnecessary - we already install / use buildx```

We need to setup buildx with qemu to build and push for multiple architecture. If we do not set up buildx before building with `--push` flag, then it fails with below error: 

![image](https://user-images.githubusercontent.com/40816837/157629780-892289ee-e701-409d-9d29-43d6ea487b0f.png)